app/src/main/java/org/metabrainz/mobile/util/SingleLiveEvent.kt

```diff
@@ -1,18 +1,3 @@
-/*
```

Comment:
Right. This is interesting. Btw we do not use this class anywhere. Do you want me to remove it because I don't think we will be using this anywhere in the future?

Reply:
Since we aren't using this anywhere, I am deleting this class. It will be present in the git history in case we need it for future uses.